src/sagarAcharya/A19_CharacterMethods.java

```diff
@@ -0,0 +1,48 @@
+package sagarAcharya;
+
+import java.util.Scanner;
+
+class A19_CharacterMethods{
+	
+	public static void main(String[] args){
+		A19_CharacterMethods a19 = new A19_CharacterMethods();
+		/* Scanner sc = new Scanner(System.in);
+		System.out.println("Enter your input : ");
+		String input = sc.next(); */
+		a19.printMethodDetails("TE1ch2noC2red9iTs");
+	}
+	
+	void printMethodDetails(String input){
+		int digitCount = 0;
```

Comment:
You can right this code in one line too.
Like int digitCount =0, uppercount =0, lowerCount =0;